custom/icds/management/commands/migrate_pse_attendance.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import print_function
+
+from django.core.management.base import BaseCommand
+
+from django.db import connections
+
+
+new_table = 'config_report_icds-cas_static-child_cases_monthly_v2_198ccc06'
+old_table = 'config_report_icds-cas_static-child_cases_monthly_tabl_551fd064'
```

Comment:
Is this pulling data from a tableau data source to a non tableau data source. I think tha't sfine, just want to make sure it's intentional